tensorbay/cli/utility.py

```diff
@@ -181,3 +181,14 @@ def _clean_up(editor_input: Optional[str]) -> Tuple[str, str]:
     if not cleaned_up_lines:
         return "", ""
     return cleaned_up_lines[0].lstrip(), "\n".join(cleaned_up_lines[1:])
+
+
+def errors_handler(err_message: str) -> NoReturn:
+    """Handling errors properly in command line.
```

Comment:
No need to use "ing".
And the message is not accurate.
Just describe what the function does in one sentence.